azure/packages/external-controller/src/app.ts

```diff
@@ -37,18 +32,11 @@ const azureUser = {
     additionalDetails: userDetails,
 };
 
-const connectionConfig: AzureRemoteConnectionConfig | AzureLocalConnectionConfig = useAzure
-    ? {
-          type: "remote",
-          tenantId: "",
-          tokenProvider: new AzureFunctionTokenProvider("", azureUser),
-          endpoint: "",
-      }
-    : {
```

Comment:
You could bring AzureFunctionTokenProvider class to this example (entire file/class) and consume it here. Then we can use the example/docs to point to DCOM docs.